src/metrics/gauge.rs

```diff
@@ -6,12 +6,13 @@
 
 use std::sync::atomic::{AtomicIsize, Ordering};
 use std::sync::Arc;
+use std::fmt::Display;
 
 /// Naive implementation of a `Gauge`.
 #[derive(Debug)]
 pub struct StdGauge {
     /// The gauge value.
-    value: AtomicIsize,
+    pub value: AtomicIsize,
```

Comment:
Not at present, I have another PR that will utilize that, a bit of leakage on my part